### PR TITLE
Dockerfile for quick demo/troubleshooting purpose.

### PR DESCRIPTION
**Pull request by lalitb**
I have been using this Dockerfile for some time as an environment for quick troubleshooting and felt it would act as a good playground for new devs/users to quickly try out opentelemetry-cpp along with exporters ( OTLP, Jaeger, Zipkin)

Also, we can further explore if this can be integrated into CI/CD pipeline ( ie running workflows on docker container instead of directly on github hosted virtual machines)

### CODE REVIEW

File: docker/ubuntuLatest/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+        -DgRPC_BUILD_TESTS=OFF \
+        -DgRPC_ABSL_PROVIDER=package \
+        -DgRPC_SSL_PROVIDER=package \
+        .. && \
+    make -j${nproc} install && make clean && ldconfig && \
+    cd ../..
+
+#install thrift
+ARG THRIFT_VERSION=0.14.1
+RUN apt-get install -y --no-install-recommends \
```
Comment (ThomsonTan) [inline docker/ubuntuLatest/Dockerfile:26]: Could we invoke the `setup_*` scripts to create the container instead of duplicating the preparation steps?

Comment (lalitb) [reply docker/ubuntuLatest/Dockerfile:26]: Yes agree, this can be simplified. Will change it.

**Comment by lalitb** [comment]
incorrectly closed by bot.

**Comment by ThomsonTan** [comment]
Could we merge this as it is?

**Comment by lalitb** [comment]
> Could we merge this as it is?

I will fix it this week for comments, to be ready for merge.

**Comment by lalitb** [comment]
> > Could we merge this as it is?
> 
> I will fix it this week for comments, to be ready for merge.

I think we are good to merge this. As discussed in the community meeting today, @esigo will be making further changes on this.

**Review by ThomsonTan — APPROVED** [review]

**Pull request merged**